### PR TITLE
Add release notes for 1.31.3

### PR DESCRIPTION
**Pull request by ikhoon**

### CODE REVIEW

**Comment by ikhoon** [comment]
image

**Review by trustin — APPROVED** [review]
:bow:

**Review by jrhee17 — APPROVED** [review]

**Comment by ikhoon** [comment]
Changelog: https://github.com/line/armeria/compare/armeria-1.31.2...patch_1.31.3?expand=1

**Review by minwoox — APPROVED** [review]
👍

File: site/src/pages/release-notes/1.31.3.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+date: 2024-12-11
+---
+
+## 🛠️ Bug fixes
+
+- <type://ResponseTimeoutMode#FROM_START> correctly bounds requests that go through <type://RetryingClient> #6025
```
Comment (minwoox) [inline site/src/pages/release-notes/1.31.3.mdx:7]:
```suggestion
- <type://ResponseTimeoutMode#FROM_START> correctly bounds requests that go through <type://RetryingClient>. #6025
```

File: site/src/pages/release-notes/1.31.3.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- Fix the bug where `EventLoop` is blocked for a long time by <type://ConnectionPoolListener#metricCollecting(MeterRegistry)>
+  when a connection is closed. #6024
+- <type://KubernetesEndpointGroup> automatically reconnects a `Watcher` when `WatcherException` is raised. #6023
```
Comment (minwoox) [inline site/src/pages/release-notes/1.31.3.mdx:10]:
```suggestion
- <type://KubernetesEndpointGroup> automatically reconnects when a `WatcherException` is raised from a `Watcher`. #6023
```

**Pull request merged**